src/leases/components/leaseSections/rent/BasisOfRentEdit.js

```diff
@@ -1069,6 +1097,7 @@ class BasisOfRentEdit extends PureComponent<Props, State> {
                   fieldAttributes={getFieldAttributes(leaseAttributes, LeaseBasisOfRentsFieldPaths.TYPE)}
                   onChange={this.onChangeTypeOptions}
                   name={`${field}.type`}
+                  disabled={!!savedBasisOfRent && !!savedBasisOfRent.locked_at}
```

Comment:
This is repeated quite a few times in here; could just be evaluated once somewhere higher up and then that variable used in each occurrence instead.

Reply:
Yep, there's a lot of repetitive stuff in this file. Let's start from somewhere and I'll create a variable out of this 👍